R/random_genotype_matrix.R

```diff
@@ -1,6 +1,6 @@
 #' Random Genotype Matrix.
 #'
-#' A dataset containing 5747 samples with 2938 SNPs.
+#' A dataset containing 2938 samples with 5747 SNPs.
```

Comment:
Is this right? With tidy data shouldn't the rows be samples and the columns be features? Not sure what we had before

Reply:
That is the case. You are right though, the description sounds like samples are rows. Will change that